docs/features/multiple-networks/multi-homing.md

```diff
@@ -167,6 +167,9 @@ spec:
   IP addresses in a `ipamclaims.k8s.cni.cncf.io` object. This IP addresses will
   be reused by other pods if requested. Useful for KubeVirt VMs. Only makes
   sense if the `subnets` attribute is also defined.
+- `physicalNetworkName` (string, optional): the name of the physical network to
+  which the OVN overlay will connect. When omitted, it will default to the value
+  of the localnet network `name`.
```

Comment:
This is in the `layer2` topology section, where it doesn't make sense. 

It should be moved to the localnet topology section instead.

Reply:
[Fixed](https://github.com/ovn-kubernetes/ovn-kubernetes/compare/24b8eaa35e3976df721fceb39811ac5927bb531e..f5be98de94e31e5354d2972cc6fc8c797a5e97d6), sorry for the mistake